aicsimageio/readers/reader.py

```diff
@@ -198,18 +265,20 @@ def shape(self) -> Tuple[int]:
         shape: Tuple[int]
             Tuple of the image array's dimensions.
         """
-        pass
+        return self.xarray_dask_data.shape
```

Comment:
how do you guarantee that this shape is populated in the xarray_dask_data property?

Reply:
This is one of those delayed properties. So by simply using `xarray_dask_data` property, if `_xarray_dask_data` isn't populated, it will populate.

I.e. this will either be populated, or, _will populate_ `xarray_dask_data`